include/swift/Strings.h

```diff
@@ -121,6 +121,12 @@ constexpr static BuiltinNameStringLiteral BUILTIN_TYPE_NAME_SILTOKEN = {
 constexpr static BuiltinNameStringLiteral BUILTIN_TYPE_NAME_WORD = {
     "Builtin.Word"};
 
+/// The name of the semantics of make utf8 string
```

Comment:
The string passed as a semantics attribute to signal to the optimizer that a function is a call functionally to string.makeUTF8?

Reply:
That's better :)